### PR TITLE
[Refactor][doc] add doc for auth management and conf

### PR DESCRIPTION
**Pull request by JNSimba**
## Problem Summary:

1. add config for be/fe
2. add authority management
3. add advance en doc / broker doc /resource doc

## Checklist(Required)

1. Does it affect the original behavior: (Yes/No/I Don't know)
4. Has unit tests been added: (Yes/No/No Need)
5. Has document been added or modified: (Yes/No/No Need)
6. Does it need to update dependencies: (Yes/No)
7. Are there any changes that cannot be rolled back: (Yes/No)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: new-docs/zh-CN/advanced/broker.md [hunk 1]
```diff
@@ -24,4 +24,205 @@ specific language governing permissions and limitations
...
+
+## 支持的存储系统
+
+不同的 Broker 类型支持不同的存储系统。
+
+1. 社区版 HDFS
+   - 支持简单认证访问
+   - 支持通过 kerberos 认证访问
+   - 支持 HDFS HA 模式访问
+2. 百度 HDFS/AFS（开源版本不支持）
```
Comment (hf200012) [inline new-docs/zh-CN/advanced/broker.md:63]: Delete this information in Baidu community and directly introduce the version supported

File: new-docs/zh-CN/admin-manual/user-privilege.md [hunk 2]
```diff
@@ -24,4 +24,192 @@ specific language governing permissions and limitations
...
+
+   在这种场景下，可以为管理员赋予 ADMIN 权限或 GRANT 权限。对 RD 赋予对任意或指定数据库表的 CREATE、DROP、ALTER、LOAD、SELECT 权限。对 Client 赋予对任意或指定数据库表 SELECT 权限。同时，也可以通过创建不同的角色，来简化对多个用户的授权操作。
+
+2. 场景二
+
+   一个集群内有多个业务，每个业务可能使用一个或多个数据。每个业务需要管理自己的用户。在这种场景下。管理员用户可以为每个数据库创建一个拥有 DATABASE 层级 GRANT 权限的用户。该用户仅可以对用户进行指定的数据库的授权。
+
+3. 黑名单
+
+   Doris 本身不支持黑名单，只有白名单功能，但我们可以通过某些方式来模拟黑名单。假设先创建了名为 `user@'192.%'` 的用户，表示允许来自 `192.*` 的用户登录。此时如果想禁止来自 `192.168.10.1` 的用户登录。则可以再创建一个用户 `cmy@'192.168.10.1'` 的用户，并设置一个新的密码。因为 `192.168.10.1` 的优先级高于 `192.%`，所以来自 `192.168.10.1` 将不能再使用旧密码进行登录。
```
Comment (hf200012) [inline new-docs/zh-CN/admin-manual/user-privilege.md:215]: Add more help documentation, link to SQL GRANTS documentation

File: new-docs/zh-CN/advanced/broker.md (hunk 1 above)
Comment (JNSimba) [reply new-docs/zh-CN/advanced/broker.md:63]: thanks ,already modified

**Review by hf200012 — APPROVED** [review]
LGTM

**Pull request merged**